e2e/portfolio-projects/test_portfolio_projects_crud.py

```
@@ -0,0 +1,402 @@
+            print("\nScreenshots saved to /tmp/test_portfolio_*.png")
+
+            return True
+
+        except AssertionError as e:
+            print(f"\n[ASSERTION ERROR] {e}")
+            take_screenshot(page, "portfolio_error_assertion", "Assertion error")
+            import traceback
+
+            traceback.print_exc()
+            return False
+        except Exception as e:
```

Ruff (0.14.4)

Do not catch blind exception: `Exception`

(BLE001)